lib/src/model/auth_bool_field.dart

```diff
@@ -0,0 +1,48 @@
+// SPDX-FileCopyrightText: 2022 Iconica
+//
+// SPDX-License-Identifier: BSD-3-Clause
+
+import 'package:flutter/material.dart';
+import 'package:flutter_input_library/flutter_input_library.dart';
+import 'package:flutter_registration/flutter_registration.dart';
+
+class AuthBoolField extends AuthField {
+  AuthBoolField({
+    required super.name,
+    required this.widgetType,
+    super.title,
+    super.validators = const [],
+    super.value = '',
```

Comment:
super.value = false
(this '' is causing a bug that i fixed in TU by setting the value to false)